e2e/tests/datasets/dataset-create.spec.ts

```diff
@@ -138,24 +138,24 @@ describe('Create Dataset', () => {
 
   test('Workspace READER cannot edit dataset', async () => {
     // READER log in in new Incognito page.
-    const newPage = await signInAs(config.readerUserName, config.userPassword);
+    await signInWithAccessToken(page, config.readerUserAccessTokenFilename);
 
     // Find workspace created by previous test. If not found, test will fail.
-    const workspaceCard = await findWorkspaceCard(newPage, workspace);
+    const workspaceCard = await findWorkspaceCard(page, workspace);
     await workspaceCard.clickWorkspaceName(true);
 
-    const readerDataPage = new WorkspaceDataPage(newPage);
+    const readerDataPage = new WorkspaceDataPage(page);
     await readerDataPage.waitForLoad();
 
     // Create Cohorts and Datasets button are disabled.
     expect(await readerDataPage.getAddDatasetButton().isCursorNotAllowed()).toBe(true);
     expect(await readerDataPage.getAddCohortsButton().isCursorNotAllowed()).toBe(true);
 
     await readerDataPage.openDatasetsSubtab();
-    await waitWhileLoading(newPage);
```

Comment:
what is the difference between page with newPage? Shall I create new one?

Reply:
test passing without creating new page.

Reply:
I already have a PR to fix it too. [https://github.com/all-of-us/workbench/pull/5148/files](https://github.com/all-of-us/workbench/pull/5148/files) 
Keep your changes, and I'll undo it in my PR.